wordpress-mobile/WordPress-iOS
# Update screenshot tooling

This PR updates the screenshot tooling in order to handle the new iPhone Xs Max size. A new background image that considers the iPhone X generation "notch" is included as well.
The PR also updates the promo messages files to include the latest translations.

To Test:
1. Setup the Screenshot generation UI test
2. Run `bundle exec fastlane snapshot`
3. Verify that all the screenshots are generated, included the Xs Max ones.
4. Run `bundle exec fastlane create_promo_screenshots`
5. Verify that all the promo screenshots are correctly generated.
